base/strings/string_util_starboard.h

```diff
@@ -25,13 +25,9 @@
 
 namespace base {
 
-#if defined(vsnprintf)
-#undef vsnprintf
-#endif
-
 inline int vsnprintf(char* buffer, size_t size,
                      const char* format, va_list arguments) {
-  return SbStringFormat(buffer, size, format, arguments);
+  return ::vsnprintf(buffer, size, format, arguments);
```

Comment:
We should include stdio.h in SB 16+ for this file

Reply:
Fixed.

Reply:
It was? stdio.h is still included in string_util.h but not in this file from what I can see

Reply:
Added the inline function and missed the header. My bad.

Reply:
Great, thank you